angular/projects/spark-angular/src/lib/components/sprk-award/sprk-award.component.ts

```diff
@@ -59,148 +55,183 @@ import { Component, Input } from '@angular/core';
+   * disclaimer.
    */
   @Input()
   analyticsStringDisclaimer: string;
   /**
-   * Expects a space separated string
-   * of classes to be added to the
+   * The string that will be assigned to the `data-analytics` attribute of
+   * the clickable disclaimer.
+   */
+  @Input()
+  disclaimerAnalyticsString: string;
+  /**
+   * Expects a space separated string of classes to be added to the
    * component.
    */
   @Input()
   additionalClasses: string;
+  // TODO - Remove as part of Issue 1279
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * first image.
+   * Deprecated - use `imgOneAdditionalClasses` instead. Expects a space
+   * separated string of classes to be added to the first image.
    */
   @Input()
   additionalClassesImgOne: string;
+  // TODO - Remove as part of Issue 1279
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * second image.
+   * Deprecated - use `imgTwoAdditionalClasses` instead. Expects a space
+   * separated string of classes to be added to the second image.
    */
   @Input()
   additionalClassesImgTwo: string;
   /**
-   * The text that appears above the
-   * images and serves as the heading
-   * for the Award.
+   * Expects a space separated string of classes to be added to the first
+   * image.
+   */
+  @Input()
+  imgOneAdditionalClasses: string;
+  /**
+   * Expects a space separated string of classes to be added to the
+   * second image.
+   */
+  @Input()
+  imgTwoAdditionalClasses: string;
+  // TODO - Remove as part of Issue 1279
+  /**
+   * Deprecated - use `heading` instead. The text that appears above the
+   * images and serves as the heading for the Award.
    */
   @Input()
   title: string;
   /**
-   * The text that will be the clickable
-   * title of the disclaimer toggle.
+   * The text that appears above the images and serves as the heading
+   * for the Award.
+   */
+  @Input()
+  heading: string;
+  /**
+   * The text that will be the clickable title of the disclaimer toggle.
    */
   @Input()
   disclaimerTitle: string;
   /**
-   * The text that will be inside the
-   * disclaimer toggle.
+   * The text that will be inside the disclaimer toggle.
    */
   @Input()
   disclaimerCopy: string;
   /**
-   * If `false`, the disclaimer
-   * toggle will not be rendered.
+   * If `false`, the disclaimer toggle will not be rendered.
    */
   @Input()
   disclaimer: string;
   /**
-   * The value supplied will be assigned
-   * to the `data-id` attribute on the
-   * component. This is intended to be
-   * used as a selector for automated
-   * tools. This value should be unique
-   * per page.
+   * The value supplied will be assigned to the `data-id` attribute on the
+   * component. This is intended to be used as a selector for automated tools.
+   * This value should be unique per page.
    */
   @Input()
-  idString: string;
+  idString: string = null;
```

Comment:
Hey @bobwise why did we set this to null here?

Reply:
We don't need this anymore. At one point the component was always rendering out `data-id` even if it didn't have a value.